doc/Core.xml

```diff
@@ -1102,9 +1102,9 @@
       <para>A successful discovery provides the device service address. Once a client has the device service address it can receive detailed device information through the device service, see section  <xref linkend="_Toc251333295" /> below.</para>
     </section>
     <section>
-      <title>Profiles</title>
-      <para>Device functionality can be grouped to so called profiles. The profiles themselves are defined in separate specifications. </para>
-      <para>For each profile a number of services and functions are mandatory which are defined in the respective specifications.</para>
+      <title>Profiles and Addons</title>
+      <para>Device functionality can be grouped to so called profiles and addons. Profles and addons themselves are defined in separate specifications. </para>
+      <para>For each profile or addon, a number of services and functions are mandatory which are defined in the respective specifications.</para>
```

Comment:
I figured it was better to combine these sections then leave separate with basically the same content.

Reply:
Typo: Profles -> Profiles

Reply:
It would also be better to mention the key difference between a Profile and Addon in the same section, like AddOn depends/complements any existing profile.

Reply:
Changed based on feedback.